tool/dart_dev/config.dart

```diff
@@ -12,19 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-library tool.dev;
+import 'package:dart_dev/dart_dev.dart';
 
-import 'package:dart_dev/dart_dev.dart' show dev, config;
-
-main(List<String> args) async {
-  // https://github.com/Workiva/dart_dev
-
-  List<String> directories = ['lib/', 'test/', 'tool/'];
-
-  config.analyze.entryPoints = ['lib/', 'test/', 'test/unit/', 'tool/'];
-  config.copyLicense.directories = directories;
-  config.format.paths = directories;
-  config.test.unitTests = ['test/unit/'];
-
-  await dev(args);
-}
+final Map<String, DevTool> config = coreConfig;
```

Comment:
Since we just use the `coreConfig` unchanged, we could just delete this file entirely.